tests/js/route.test.js

```diff
@@ -502,8 +564,9 @@ describe('route()', () => {
 
     test("can append 'extra' string/number parameter to query", () => {
         // 'posts.index' has no parameters
-         same(route('posts.index', 'extra'), 'https://ziggy.dev/posts?extra=');
-         same(route('posts.index', 1), 'https://ziggy.dev/posts?1=');
+        same(route('posts.index', 'extra'), 'https://ziggy.dev/posts?extra=');
+        same(route('posts.index', {extra: 2}), 'https://ziggy.dev/posts?extra=2');
```

Comment:
I added this test because prior to this PR, this would result in 'https://ziggy.dev/posts?[Object object]=' which is a bug fixed on line https://github.com/tighten/ziggy/pull/514/files#diff-372121aa5f6bd350905f140546c9ea2736d4b4f2a6cf987115c77eba82f936ddR217

Reply:
route('name', [{ extra: 2 }]) and route('name', { extra: 2 }) should both give the same result (before PR and after PR) except before they both would give ?[Object object]=